icu4c/source/test/intltest/numbertest_affixutils.cpp

```diff
@@ -25,7 +25,7 @@ class DefaultSymbolProvider : public SymbolProvider {
             case TYPE_PLUS_SIGN:
                 return fSymbols.getConstSymbol(DecimalFormatSymbols::ENumberFormatSymbol::kPlusSignSymbol);
             case TYPE_APPROXIMATELY_SIGN:
-                return fSymbols.getConstSymbol(DecimalFormatSymbols::ENumberFormatSymbol::kApproximatelySignSymbol);
+                return u"≃";
```

Comment:
I always thought that the approximately equals sign is ≈. And Latex and the [interwebs/Wikipedia](https://en.wikipedia.org/wiki/Approximation#Unicode) concurs.

Reply:
Acknowledged; the intent of this test is to return some "interesting" symbol to make sure that the symbol gets passed through the stack.  The actual locale data is determined by translators.